src/lib/registerServiceWorker.js

```diff
@@ -64,8 +66,8 @@ function checkValidServiceWorker(swUrl) {
         registerValidSW(swUrl);
       }
     })
-    .catch(() => {
-      console.log('No internet connection found. App is running in offline mode.');
+    .catch(err => {
+      ErrorPopup('No internet connection found. App is running in offline mode.', err);
     });
```

Comment:
I don't think we should show errorPopup for this file, I don't think it is critical for the user